File: 240307/상품 코드/product-code.py
class Product:
  def __init__(self, name, code):
    self.name = name
    self.code = code

product1 = Product('codetree',50)
args = tuple(input().split())
product2 = Product(*args)

print(f'product {product1.code} is {product1.name}')
print(f'product {product2.code} is {product2.name}')

File: 240424/상품 코드/product-code.js
class Product {
    constructor(name, code) {
        this.name = name;
        this.code = code;
    }
}

const fs = require('fs')
const input = fs.readFileSync(0).toString().trim().split(' ')

const product1 = new Product("codetree", "50")
const product2 = new Product(input[0], input[1])

console.log(`product ${product1.code} is ${product1.name}`)
console.log(`product ${product2.code} is ${product2.name}`)